sbin/jtreg.sh

```diff
@@ -92,7 +92,7 @@ settingUpEnvironmentVariablesForJTREG()
 runJtregViaMakeCommand()
 {
   echo "Running jtreg via make command (debug logs enabled)"
-  if [ -z $JTREG_TEST_SUBSETS }; then
+  if [ -z "$JTREG_TEST_SUBSETS" ]; then
     make test jobs=10 LOG=debug
```

Comment:
Isn't the standard just to use `$VAR` normally, and `${VAR}` when you need to (e.g. `${VAR}SOME OTHER STRING` or `${VAR:-default}`?